src/runner.rs

```diff
@@ -191,13 +194,16 @@ pub mod platform {
     use winreg::enums::*;
 
     pub fn firefox_default_path() -> Option<PathBuf> {
+        if let Some(path) = find_binary("firefox.exe") {
+            return Some(path)
+        }
         let opt_path = firefox_registry_path().unwrap_or(None);
```

Comment:
It would be good to have consistent behavior between platforms, but I do see your point. My use case is switching between running Selenium tests against Firefox release, aurora, and nightly without hard-coding these paths. I'm using macOS, so removing this code for Windows wouldn't affect me personally. Let me know what you'd like me to do.